src/IssueViz.Security/Taint/TaintStore.cs

```diff
@@ -52,6 +55,15 @@ internal sealed class TaintStore : ITaintStore
         public IReadOnlyCollection<IAnalysisIssueVisualization> GetAll() => taintVulnerabilities;
         
         public AnalysisInformation GetAnalysisInformation() => analysisInformation;
+        public void Add(IAnalysisIssueVisualization taintIssue)
```

Comment:
This will be replaced with real implementation in https://github.com/SonarSource/sonarlint-visualstudio/issues/3678